src/components/input/bl-input.css

```
@@ -84,6 +89,16 @@ input {
   background-color: transparent;
 }
 
+input:disabled {
+  cursor: not-allowed;
+}
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus {
+  box-shadow: 0 0 0 40rem var(--autofill-bg-color) inset;
```

I see that it's not possible to just set background-color because of some browsers doesn't allow it. Since this is weird to see for future developers, I have 2 suggestions:

1. Use `background-color` anyway. It'll work for the browsers that allows setting it.
2. As an addition to using `background-color` add this box-shadow trick with a comment on top of it.

```suggestion
  background-color: var(--autofill-bg-color);
  
  /**
   * Some browsers doesn't allow setting background-color
   * https://developer.mozilla.org/en-US/docs/Web/CSS/:autofill
   */
  box-shadow: 0 0 0 40rem var(--autofill-bg-color) inset;
```